index.md

```diff
@@ -6,12 +6,12 @@ root: .
 Data Carpentry’s aim is to teach researchers basic concepts, skills, and tools for working
 with data so that they can get more done in less time, and with less pain. This workshop
 teaches data management and analysis for genomics research including: 
-best practices for organization of bioinformatics projects and data, use of command line 
-utilities, use of command line tools to analyze sequence quality and 
+best practices for organization of bioinformatics projects and data, use of command-line 
```

Comment:
the "and" is needed here because it is "use of command line tools to analyze sequence quality and perform variant calling" and then as a separate concept "connecting to and using cloud computing". I'll make this one change and then merge the PR. Thanks you @anwarMZ